src/from_markdown.js

```diff
@@ -224,8 +224,8 @@ export const defaultMarkdownParser = new MarkdownParser(schema, markdownit("comm
   bullet_list: {block: "bullet_list"},
   ordered_list: {block: "ordered_list", getAttrs: tok => ({order: +tok.attrGet("start") || 1})},
   heading: {block: "heading", getAttrs: tok => ({level: +tok.tag.slice(1)})},
-  code_block: {block: "code_block"},
-  fence: {block: "code_block", getAttrs: tok => ({params: tok.info || ""})},
+  code_block: {block: "code_block", noOpenClose: true},
+  fence: {block: "code_block", getAttrs: tok => ({params: tok.info || ""}), noOpenClose: true},
```

Comment:
The options in here are not actually necessary, but help if people are using this as an example for extensions.